app/controllers/admin/questions_controller.rb

```diff
@@ -0,0 +1,55 @@
+class Admin::QuestionsController < AdminController
+  def index
+    @bank_questions = Question.for_banks
+    @partner_questions = Question.for_partners
+  end
+
+  def new
+    @question = Question.new
+  end
+
+  def create
+    @question = Question.create(question_params)
+    if @question.valid?
```

Comment:
```suggestion
    if @question.save
```
The insert/update actions on an active record model automagically call "valid?" :). I prefer to use the `save` or `update` action and catch whether that failed or not. Depending on your system, a valid can trigger one set of  validations on create vs on update. For that reason, I you'll want to validate against the actual action.

Reply:
I refactored it the way that you recommended.